src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+

```

Comment:
<img width="1013" alt="Screenshot 2022-01-17 at 08 58 18" src="https://user-images.githubusercontent.com/33026471/149723241-7f281521-7461-4dab-9289-7410b856b134.png">
Let's remove redundant empty lines here and in main

Reply:
Mistakes were corrected